doc/man7/EVP_PKEY-DH.pod

```diff
@@ -17,15 +17,10 @@ See L<EVP_PKEY-FFC(7)> for more information about FFC keys.
 
 The B<DH> key type uses PKCS#3 format which saves p and g, but not the 'q' value.
 The B<DHX> key type uses X9.42 format which saves the value of 'q' and this
-must be used for FIPS186-4.
+must be used for FIPS186-4. If key validation is required users should be aware
+of the problems associated with FIPS186-4 style parameters as discussed in
```

Comment:
I don't think _problems_ is a good word here?
What about _nuances_, _details, _specifics_ or _subtleties_ instead?